frontend/src/app/shared/components/profile-menu/profile-menu.component.html

```diff
@@ -1,8 +1,12 @@
-<div class="user-info">
-    <app-avatar class="avatar" [userName]="'John Doe'" size="36"></app-avatar>
+<div class="user-info" *ngIf="currentUser">
+    <app-avatar
+        class="avatar"
+        [userName]="currentUser.firstName"
```

Comment:
just a quick suggestion, please add a first and last name to userName param, as avatar component can generate more diverse images with these params